examples/confluent_cloud.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This is a simple example demonstrating how to produce a message to
+# Confluent Cloud then read it back again.
+#
+# https://www.confluent.io/confluent-cloud/
+#
+# Confluent Cloud does not auto-create topics. You will need to use the ccloud
+# cli to create the python-test-topic topic before running this example. The
+# <ccloud bootstrap servers>, <ccloud key> and <ccloud secret> parameters are
+# available via the confluent cloud web interface. For more information,
+# refer to the quick-start:
+#
+# https://docs.confluent.io/current/cloud-quickstart.html
+#
+# to execute using Python 2.x:
+#   virtualenv ccloud_example
+#   source ccloud_example/bin/activate
+#   pip install confluent_kafka
+#   python example.py
```

Comment:
the file is called examples/confluent_cloud.py  (please verify that instructions work)

Reply:
ahh. it's called that in my examples folder, which is where I copied this in from...